Integrations/MicrosoftGraphListener/MicrosoftGraphLIstener.py

```diff
@@ -0,0 +1,982 @@
+        parsed_draft = MsGraphClient._parse_item_as_dict(created_draft)
+        human_readable = tableToMarkdown(f'Created draft with id: {parsed_draft.get("ID", "")}', parsed_draft)
+        ec = {self.CONTEXT_DRAFT_PATH: parsed_draft}
+
+        return human_readable, ec, created_draft
+
+    def send_email(self, **kwargs):
+        """
+        Sends email from user's mailbox, the sent message will appear in Sent Items folder
+        """
+        suffix_endpoint = f'/users/{self._mailbox_to_fetch}/sendMail'
+        message_content = MsGraphClient._build_message(**kwargs)
+        self._http_request('POST', suffix_endpoint, json_data={'message': message_content},
+                           resp_type="text")
+
+        message_content.pop('attachments', None)
+        message_content.pop('internet_message_headers', None)
+        human_readable = tableToMarkdown(f'Email was sent successfully.', message_content)
+        ec = {self.CONTEXT_SENT_EMAIL_PATH: message_content}
+
+        return human_readable, ec
+
+    def reply_to(self, to_recipients, comment, message_id):
+        """
+        Sends reply message to recipients.
+
+        :type to_recipients: ``list``
+        :param to_recipients: List of recipients to reply.
+
+        :type comment: ``str``
+        :param comment: The comment to send as a reply
+
+        :type message_id: ``str``
+        :param message_id: The message id to reply.
+
+        :return: String representation of markdown message regarding successful message submission.
+        rtype: ``str``
+        """
+        suffix_endpoint = f'/users/{self._mailbox_to_fetch}/messages/{message_id}/reply'
+        reply = MsGraphClient._build_reply(to_recipients, comment)
+        self._http_request('POST', suffix_endpoint, json_data=reply, resp_type="text")
+
+        return f'### Replied to: {", ".join(to_recipients)} with comment: {comment}'
+
+    def send_draft(self, draft_id):
+        """
+        Send draft message.
+
+        :type draft_id: ``str``
+        :param draft_id: Draft id to send.
+
+        :return: String representation of markdown message regarding successful message submission.
+        :rtype: ``str``
+        """
+        suffix_endpoint = f'/users/{self._mailbox_to_fetch}/messages/{draft_id}/send'
+        self._http_request('POST', suffix_endpoint, resp_type="text")
+
+        return f'### Draft with: {draft_id} id was sent successfully.'
+
+    def test_connection(self):
+        """
+        Basic connection test instead of test-module.
+
+        :return: Returns markdown string representation of success or Exception in case of login failure.
+        rtype: ``str`` or Exception
+        """
+        suffix_endpoint = f'users/{self._mailbox_to_fetch}'
+        user_response = self._http_request('GET', suffix_endpoint)
+
+        if user_response.get('mail') != '' and user_response.get('id') != '':
+            return_outputs('```✅ Success!```')
```

Comment:
🥇 
does it look good in the war-room?

Reply:
Looks amazing in war-room
<img width="1550" alt="test" src="https://user-images.githubusercontent.com/45535078/68680916-234b5300-056b-11ea-8286-189165d0f042.png">

Reply:
was this tested in both 4.5 and 5.0?

Reply:
yes, worked both in 4.5 and 5.0